.github/workflows/linters.yml

```diff
@@ -0,0 +1,20 @@
+name: Linters
```

Comment:
- [ ] It would be better if our README file is descriptive of the project. For example; let us have a title of the project with the necessary description as well as instructions about how to clone and run the app locally